lib/auth/github.go

```diff
@@ -163,12 +185,12 @@ func checkGithubOrgSSOSupport(ctx context.Context, conn types.GithubConnector, u
 
 // orgUsesExternalSSO returns true if the Github organization org
 // uses external SSO.
-func orgUsesExternalSSO(ctx context.Context, org string, client httpRequester) (bool, error) {
+func orgUsesExternalSSO(ctx context.Context, endpointURL, org string, client httpRequester) (bool, error) {
```

Comment:
`endpointURL` is not a great name for this variable, since it's not actually a URL. If it was a URL you wouldn't need to prepend it with `https://`

Reply:
Changed so it's actually a URL